db/migrations/0003.created_date_datatype_change.py

```diff
@@ -0,0 +1,8 @@
+#
+# file: migrations/0003.created_date_datatype_change.py
+#
+from yoyo import step
+
+__depends__ = {'0002.add_col_published_date'}
+
+step("ALTER TABLE course MODIFY column created_at DATETIME NULL")
```

Comment:
Changes the `created_at` data type to be consistence with `published_at` date

Reply:
Yeah, this change makes sense to me. I was also looking at `DATETIME`. I'm OK including this here, though I thought we were talking about doing this as a separate issue.